modules/storages/spec/features/create_file_links_spec.rb

```diff
@@ -50,7 +50,7 @@
   let(:sync_service) do
     sync_service = instance_double(Storages::FileLinkSyncService)
     allow(sync_service).to receive(:call) do |file_links|
-      ServiceResult.success(result: file_links.each { |file_link| file_link.origin_permission = :view })
+      ServiceResult.success(result: file_links.each { |file_link| file_link.origin_status = :view_allowed })
```

Comment:
Wow, didn't know `.each` is returning the array. Feels weird. I mean yeah, for changing the object this is a good method.